src/c4group_ng.cpp

```diff
@@ -645,7 +645,7 @@ int main(int argc, char *argv[])
 		std::println("          -v View  -l List  -d Delete  -r Rename  -s Sort");
 		std::println("          -p Pack  -u Unpack  -x Explode");
 		std::println("          -k Print maker");
-		std::println("          -g [source] [target] [title] Make update");
+		std::println("          -g[a] [source] [target] [title] Make update [allow missing target]");
```

Comment:
“Make update [and allow…” or “Make update [allowing…”
Preferably also extend the description to say “missing target group when applying update”